pkg/spdk/types.go

```diff
@@ -31,6 +32,16 @@ const (
 	retryInterval = 1 * time.Second
 )
 
+const (
+	// Timeouts for RAID base bdev (replica)
+	replicaCtrlrLossTimeoutSec  = 15
```

Comment:
Could we add a comment explaining how this value is determined?

Reply:
Added. Thank you.